cpp/tensorrt_llm/batch_manager/kvCacheManager.cpp

```diff
@@ -1619,14 +1619,14 @@ std::pair<SizeType32, std::optional<KVCacheBlock::IdType>> WindowBlockManager::s
         if (pinBlocks)
         {
             searchRoot->incRefCount();
+            pinnedBlockIds.push_back(searchRoot->getBlockId());
```

Comment:
If we have a block already matched and in the search tree, do we need to pin it again?